### DISCUSSION TITLE
Changing this to work on firefox

### DISCUSSION THREAD

**Discussion by return-misaPuding** [gosoccerboy5/blacklist-pages#3] [Q&A] [answered]
What do I need to change in order to make this extension work on firefox? I want to create an extension which will allow me to copy special characters cos I'm on linux (in VirtualBox, but many things don't work on Microsloth Windoze so I prefer using linux), inputting them with Ctrl+shift+U+character ASCII value in hex doesn't work quite often and I don't want to use Chrome so I probably need to change the manifest but IDK what to change in it

**Comment by aidan-crockett** [comment, marked as answer]
I really don't know.
If you wanted to make your own extension, MDN has [their own API reference](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions) for Firefox browser extensions.
The only thing I kind of know is that Firefox extensions use the [`browser`](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API) global object instead of [`chrome`](https://developer.chrome.com/docs/extensions/reference/).

**Reply by return-misaPuding** [reply]
Thx
